server/http_test.go

```diff
@@ -236,41 +234,61 @@ func TestHTTPServer_ServeHTTP_Files2(t *testing.T) {
 	helper.Must(os.Chdir(currentDir)) // defer for error cases, would be to late for normal exit
 }
 
-func TestHTTPServer_ServeHTTP_UUID_Option(t *testing.T) {
+func TestHTTPServer_UUID_Common(t *testing.T) {
```

Comment:
Why these table tests were separated ? Replacing `server.New` with `newCouper` should be enough if this is setup related?

Reply:
After the fix (refactoring), I must test this via integration test instead of an unit test.